packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -43,6 +52,12 @@ class ErrorMessage {
   static readonly loginPortConflictTitle = "LoginPortConflict";
   static readonly loginPortConflictDescription = "Timeout waiting for port. Try again.";
   static readonly loginComponent = "login";
+  static readonly checkOnlineFailTitle = "CheckOnlineFail";
+  static readonly checkOnlineFailDetail =
```

Comment:
Why have this string while it's already in package.nls.json? As error.message, you can get default string using the one in package.nls.json.